Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -837,26 +854,35 @@ void CGameListCtrl::OnRightClick(wxMouseEvent& event)
 		if (selected_iso)
 		{
 			wxMenu popupMenu;
-			popupMenu.Append(IDM_PROPERTIES, _("&Properties"));
-			popupMenu.Append(IDM_GAME_WIKI, _("&Wiki"));
-			popupMenu.AppendSeparator();
 
-			if (selected_iso->GetPlatform() != DiscIO::IVolume::GAMECUBE_DISC)
+			if (!selected_iso->IsElfOrDol())
+			{
+				popupMenu.Append(IDM_PROPERTIES, _("&Properties"));
+				popupMenu.Append(IDM_GAME_WIKI, _("&Wiki"));
+				popupMenu.AppendSeparator();
+			}
+			if (selected_iso->GetPlatform() != DiscIO::IVolume::GAMECUBE_DISC && !selected_iso->IsElfOrDol())
 			{
 				popupMenu.Append(IDM_OPEN_SAVE_FOLDER, _("Open Wii &save folder"));
 				popupMenu.Append(IDM_EXPORT_SAVE, _("Export Wii save (Experimental)"));
 			}
 			popupMenu.Append(IDM_OPEN_CONTAINING_FOLDER, _("Open &containing folder"));
-			popupMenu.AppendCheckItem(IDM_SET_DEFAULT_ISO, _("Set as &default ISO"));
+
+			if (!selected_iso->IsElfOrDol())
+				popupMenu.AppendCheckItem(IDM_SET_DEFAULT_ISO, _("Set as &default ISO"));
 
 			// First we have to decide a starting value when we append it
 			if (selected_iso->GetFileName() == SConfig::GetInstance().m_strDefaultISO)
 				popupMenu.FindItem(IDM_SET_DEFAULT_ISO)->Check();
 
-			popupMenu.AppendSeparator();
-			popupMenu.Append(IDM_DELETE_ISO, _("&Delete ISO..."));
+			if (!selected_iso->IsElfOrDol())
+			{
+				popupMenu.AppendSeparator();
+				popupMenu.Append(IDM_DELETE_ISO, _("&Delete ISO..."));
+			}
+
 
-			if (selected_iso->GetPlatform() != DiscIO::IVolume::WII_WAD)
+			if (selected_iso->GetPlatform() != DiscIO::IVolume::WII_WAD && !selected_iso->IsElfOrDol())
```

Comment:
Since Options used to be exclusive and taken care of by this ```if```; yes, you have a point. But then again, multiple people have argued that exposing and using ```IsElfOrDol``` like that should be avoided if possible; so it is somewhat in-scope and somewhat out-of-scope.
And since you're touching this part anyways (both IsoFile and GameListCtrl here), you might aswell do it the proper way instead of hacking in something, then reworking it later.